python_transport/wirepas_gateway/keep_alive_service/keep_alive_service.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2024 Wirepas Ltd licensed under Apache License, Version 2.0
+#
+# See file LICENSE for full license details.
+#
+from enum import IntEnum
+import logging
+import struct
+from time import time, sleep
+from threading import Thread
+
+import wirepas_mesh_messaging as wmm
+
+
+KEEP_ALIVE_SERVICE_VERSION = 0x01
+KEEP_ALIVE_SRC_EP = 67
+KEEP_ALIVE_DST_EP = 67
+
+# Timeouts and periods used for the keep alive service.
+WM_MSG_RETRY_PERIOD_S = 1
```

Comment:
Is 1s enough ?

Reply:
Well if a message can't be sent to a sink 3 times in a row in ~2s, there are definitely issue with this sink. So not sending a keep alive message for that sink wouldn't be that bad.